src/lib.rs

```diff
@@ -1054,6 +1070,55 @@ where
     }
 }
 
+// borsh is byte-size little-endian de-needs-external-schema no-bit-compression serde
```

Comment:
here is simple description.

it is simpler than pb as it has no versioning or compression at all. it is very close to copy memory if you on LE machine.